terraform/apply.sh

```diff
@@ -13,6 +13,12 @@ EOF
 
 set +e
 
+if [ "<< parameters.auto_approve >>" = "true" && -n "<< parameters.target >>" ]; then
```

Comment:
I don't think thats valid bash
```suggestion
if [[ "<< parameters.auto_approve >>" = "true" && -n "<< parameters.target >>" ]]; then
```